website/src/components/sortableTable/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import Markdown from 'markdown-to-jsx';
+
+const stripMarkdown = (text) => {
+
+  let strippedText = text.replace(/\[([^\]]+)\]\([^)]+\)/g, '$1');
+
+  strippedText = strippedText.replace(/[_*`~]/g, '');
+  return strippedText;
+};
+
+const parseMarkdownTable = (markdown) => {
```

Comment:
Hey Matt! This is a totally awesome implementation. Really nice work here. It was clever to include cases for different alignment codes for table rows in markdown - I always forget those syntax options exist!

Re: what I was saying in our meeting yesterday: if you really wanted to, you could wrap `parseMarkdownTable` in a [useMemo](https://react.dev/reference/react/useMemo) hook to avoid it running on every re-render for performance reasons! But I did check out the component on the enterprise perms table, adjusted some things to re-run the state, and the performance hit is negligible. 

I wouldn't say it's a blocking change IMO, but feel free to update it if you want to! All in all, this LGTM. 🚢